client/src/lib/utils/Application/RemoveApplication.ts

```diff
@@ -1,22 +1,30 @@
-import {SupabaseClient, PostgrestError} from '@supabase/supabase-js';
-import {Profile} from '../../store/profile';
-import {createSupabaseClient} from '../../supabase/client';
+import { SupabaseClient, PostgrestError } from "@supabase/supabase-js";
+import { Profile } from "../../store/profile";
+import { createSupabaseClient } from "../../supabase/client";
 
 export async function RemoveApplication(
-    profile: Profile,
-    applicationId: string,
-    supabase: SupabaseClient = createSupabaseClient()
+  profile: Profile,
+  applicationId: string,
+  supabase: SupabaseClient = createSupabaseClient()
 ): Promise<{ success: boolean | null; error: PostgrestError | null }> {
```

Comment:
We don't need success here. We can imply that the request was successful if the error object is null since it wasn't assigned a PostgrestError value

Reply:
got it